sessions/js-array-methods-2/employees/README.md

```diff
@@ -0,0 +1,36 @@
+# JS Array Methods 2: Employees
+
+This challenge tasks you with navigating a large dataset of over 500 employees, utilizing JavaScript array methods to find out if there is any employee that fits a specific condition.
+
+## Task
+
+Take a look at the browser preview. You will see large set of index cards, one card for each employee. There are four forms that help you finding a specific employees. Unfortunately the forms do not work correct. Your task is to implement the corresponding functions correctly. See `./js/index.js` for more information.
+
```

Comment:
The forms work now! Have you pushed the solution accidentally perhaps? ^^

Reply:
No - I just wanted to help you to find a solution 😂. Removed it now.